chapters/Basics.md

```diff
@@ -317,7 +317,12 @@ let foo = "hello there!";
 ```
 
 Even though we didn't tell TypeScript that `foo` had the type `string` it was able to figure that out.
-That's a feature, and it's best not to add annotations when the type system would end up inferring the same type anyway.
+That's a feature, and it's often preferred not to add annotations when the type system would end up inferring the same type anyway.
+For example, using `Date()` without `new` could be a typo as above, but here we are explicitly saying that we expect a string:
```

Comment:
I get that this pattern *can* be used, but I'm not clear on why this clarifies the above text, or what the use-case would be for a novice user - in fact, it might confuse me more because it almost implies that TypeScript would error on the next example if I didn't add a type annotation.

Reply:
Yeah, the phrasing is not great (my fault), and the example is not great either (since I was trying to keep the change minimal).

The basic issue is that telling people to always prefer avoiding an explicit type is confusing since there are several cases in which it is a bad idea.  Some examples: when the inferred type is less precise than the intention; when code is complicated enough to make inferred types lead to unreadable type errors (common problem in ML), ... or when you want to be explicit about the type just to make code more readable (this example).